wdio.conf.js

```diff
@@ -79,7 +79,7 @@ exports.config = {
     // Define all options that are relevant for the WebdriverIO instance here
     //
     // Level of logging verbosity: trace | debug | info | warn | error | silent
-    logLevel: 'info',
+    logLevel: 'warn',
```

Comment:
Loglevel can be adjusted through the command line, I would keep the default loggin to `silent` and if needed pollute the logs with `npm run test  -- --logLevel=warn`

Reply:
perfect, was trying to make it more silent because those logs were annoying